avocado/core/test.py

```diff
@@ -139,7 +139,7 @@ def record_and_warn(*args, **kwargs):
 
         self.debugdir = None
         self.resultsdir = None
-        self.status = None
+        self.status = "ERROR"
```

Comment:
@ldoktor I like it. Indeed it seems the way to go if we want to keep the user-facing-statuses. I don't like the message `... with incorrect test status ... `. I'd rather use `...with internal test status...` or something like that, but this is minor.